### PR TITLE
Incorporate ChannelArchiver in simple data broker

### PR DESCRIPTION
**Pull request by danielballan**
Now a `search` call in the simple data broker returns data from all three sources. It returns data from "popular" sources on the channel archiver by default, plus any additional channels specified by the user.

The notion here is to return all the raw data that the user is likely to want, not all the raw data that they could possibly ask for. (That would knock over the Archiver.) The muggler layer can make this sleeker, presenting the data is a more readily usable form, giving manageable access to the many Channel Archiver sources, and requesting additional sources from the broker as needed.

Demo with additional explanation: http://nbviewer.ipython.org/gist/danielballan/418b6261f6b547c64552

### CODE REVIEW

File: databroker/broker/simple_broker.py [hunk 1]
```diff
@@ -6,12 +6,68 @@
...
+    channels = list(set(channels) | set(POPULAR_CHANNELS))
+    archiver = sources.channelarchiver.Archiver(ca_host)
+    archiver_result = archiver.get(channels, start_time, end_time,
+                                   interpolation='raw')  # never interpolate
+
+    # Format data from the Archiver like data from the events, and combine.
+    for ch_name, ch_data in zip(channels, archiver_result):
+        data += [(time, {ch_name: value}) for time, value in
+                 zip(ch_data.times, ch_data.values)]
+    return data
```
Comment (licode) [inline databroker/broker/simple_broker.py:70]: So the data is also sorted in time?

Comment (danielballan) [inline databroker/broker/simple_broker.py:70]: No. Do we want it to be?

Comment (tacaswell) [inline databroker/broker/simple_broker.py:70]: I would argue that sorting it is the job of the down-stream code (like muggler)

Comment (licode) [inline databroker/broker/simple_broker.py:70]: It is just easy to read. But if we need to pass this data to data muggler for cleaning, we can ignore this middle step.

Comment (tacaswell) [inline databroker/broker/simple_broker.py:70]: if you want it to print sorted use `sorted(res, key=lambda x: x[0])`

**Pull request merged**